src/ui/public/notify/notifier.js

```diff
@@ -310,48 +320,69 @@ Notifier.prototype._showFatal = function (err) {
  * @param  {Error|String} err
  * @param  {Function} cb
  */
-Notifier.prototype.error = function (err, cb) {
-  return add({
+Notifier.prototype.error = function (err, opts, cb) {
+  const overrideable = ['lifetime', 'icon'];
+  if (_.isFunction(opts)) {
+    cb = opts;
+    opts = {};
+  }
```

Comment:
👍 Good point about plugins, and that we're planning to re-do Notifications as a whole in the near future.